dask/dataframe/methods.py

```diff
@@ -587,6 +591,10 @@ def tolist_pandas(obj):
     return obj.tolist()
 
 
+@is_categorical_dtype_dispatch.register((pd.Series, pd.Index, pd.api.extensions.ExtensionDtype, np.dtype))
+def is_categorical_dtype_pandas(obj):
+    return pd.api.types.is_categorical_dtype(obj)
+
```

Comment:
IIUC this change itself doesn't add support for `cudf` objects, but does allow for `cudf` to start registering it's own `is_categorical_dtype_dispatch` for things like `cudf.Series`. Is that correct, or am I missing something?

Reply:
Correct, there'll be a corresponding cuDF PR that leverages this and registers the proper cuDF objects.